openedx/features/pakx/lms/overrides/views.py

```diff
@@ -493,7 +492,8 @@ class MarketingCampaignPage(AboutUsView):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.email_subject = 'Pakistan Against Workplace Harassment Form Data'
-        self.initial_data = {'message': 'Not Available. Submitted from Marketing campaign Page'}
+        self.initial_data = {'message': 'Not Available. Submitted from Marketing campaign Page',
+                             "field_to_hide": "message"}
```

Comment:
can we create a separate variable for hidden fields? and this should be a list just in case we want to hide multiple fields. i.e
```
self.hidden_fields = ['message']
```